JavaScript/chapter01/1.8 - Zero Matrix/rroque98_sol.js

```diff
@@ -0,0 +1,84 @@
+/*Write an algorithm such that if an element
+ in an M*N matrix is 0,it's entire row and column
+  are set to 0*/
+
+const zeroMatrix = (nestedArr) => {
+  var rowsAndColsWithZeros = checkForZeroIndex(nestedArr);
+  let rowLength = nestedArr[0].length;
+  let columnLength = nestedArr.length;
+  for (let i = 0; i < rowsAndColsWithZeros['rows'].length; i++) {
+    let rowIndex = rowsAndColsWithZeros['rows'][i];
+    for (let x = 0; x < rowLength; x++) {
+      nestedArr[rowIndex][x] = 0;
+    }
+  }
+  for (let i = 0; i < rowsAndColsWithZeros['columns'].length; i++) {
```

Comment:
you don't actually use `i` except for indexing into the array, so you can use the more idiomatic

```javascript
for (const colIndex of rowsAndColsWithZeroes['columns']) {
```

Same above.